oneapp-server/types/ApplicationContact.js

```diff
@@ -129,11 +129,15 @@ const typeDef = gql`
     "New Jersey Helps"
     NJHE
   }
+
+  enum State {
```

Comment:
I'm wondering if there are every situations where a non-NJ address is valid. Like could someone be a NJ resident but living with family in PA and receive the benefits at that PA address? Thinking we should define out all the states as valid in the ENUM.